contracts/contracts/BLSWallet.sol

```diff
@@ -73,6 +73,7 @@ contract BLSWallet is Initializable, IBLSWallet
     ) public onlyTrustedGateway {
         require(isZeroBLSKey(blsPublicKey), "BLSWallet: public key already set");
         blsPublicKey = blsKey;
+        pendingBLSPublicKeyTime = type(uint256).max;
```

Comment:
Also could initialise other pending times:
```
+        pendingRecoveryHashTime = type(uint256).max;
         pendingPAFunctionTime = type(uint256).max;
+        pendingBLSPublicKeyTime = type(uint256).max;
+        pendingGatewayTime = type(uint256).max;
```

Reply:
resolved in 02d64c1